x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -102,12 +132,38 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     });
 
     describe('Table Sort', () => {
-      it('sorts by rule name', async () => {
-        await table.toggleColumnSortOrFail('Rule', 'asc');
-      });
-
-      it('sorts by resource name', async () => {
-        await table.toggleColumnSortOrFail('Resource Name', 'desc');
+      type SortingMethod = (string, string) => number;
+      type SortDirection = 'asc' | 'desc';
+      // Sort by lexical order will sort by the first character of the string (case-sensitive)
+      const compareStringByLexicographicOrder = (a: string, b: string) => {
+        return +(a > b) || -(b > a);
```

Comment:
```suggestion
const compareStringByLexicographicOrder = (a: string, b: string) => {
  return a > b ? 1 : b > a ? -1 : 0;
};
```

i think this is simpler to read, and also doesn't include `-0`

Reply:
Nice!

Reply:
Done